bin/minikube/Makefile

```diff
@@ -27,7 +43,7 @@ lint-fix:  ## Run autoformatters
 .PHONY: lint-fix
 
 typecheck:  ## Run typechecking
-	poetry run mypy --show-error-codes --pretty --show-traceback .
+	poetry run mypy --show-error-codes --pretty --show-traceback . --config ../pyproject.toml
```

Comment:
Somehow `mypy` was not finding the correct config, I think it was looking at `setup.cfg`